sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClientBuilder.java

```diff
@@ -623,7 +623,7 @@ private ConnectionOptions getConnectionOptions() {
             : CbsAuthorizationType.JSON_WEB_TOKEN;
 
         return new ConnectionOptions(fullyQualifiedNamespace, credentials, authorizationType, transport, retryOptions,
-            proxyOptions, scheduler);
+            proxyOptions, scheduler, "");
```

Comment:
Why is this hardcoded to empty string?

Reply:
I told Hemant to save the EventHubs implementation one to another PR. I was trying to scope this PR. There is another open issue for Event Hubs.